stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageContent.kt

```diff
@@ -122,7 +122,10 @@ internal fun DefaultMessageContent(
         )
 
         if (message.text.isNotEmpty()) {
-            DefaultMessageTextContent(message = message)
+            DefaultMessageTextContent(
+                message = message,
+                onLongItemClick = onLongItemClick
```

Comment:
I kept our naming scheme for the sake of not deviating but I think it is not good here because:

1. `onLongItemClick` semantically says that the item is long and not the click
2. Not everything is an item, I think this should only apply to members of the Lazys

Reply:
1) I don't think that's what people think personally, `onItemLongClick` isn't much different. It can also mean once the item performs a long click if you think about it. 

2) Item can be anything really. A timestamp can be an "item" within the container.

Should be fine IMHO